Resources/Locale/en-US/supermatter/supermatter-component.ftl

```diff
@@ -0,0 +1,22 @@
+### SupermatterComponet
+
+# Used for station announcements
+supermatter-self = Supermatter
+
+# Warning message for when the Supermatter has over 50 damage and less than 700 damage
+supermatter-danger-message = Danger! Crystal hyperstructure integrity faltering! Integrity: {$integrity}%
+
+# Warning message for when the Supermatter has over 700 damage
+supermatter-warning-message = WARNING! Crystal hyperstructure integrity reaching critical levels! Integrity: {$integrity}%
+
+# Safe message for when the crystal returns to below 50 damage
+supermatter-safe-allert = Crystalline hyperstructure returning to safe operating parameters. Failsafe has been Disengaged.
+
+# Message sent once delam starts and is over the MolePenaltyThreshold
+supermatter-delamination-overmass = The Supermatter has Reached Critical Mass Falure. Singularity formation Imminent
+
+# Message sent once delam starts with no special modifiers
+supermatter-delamination-default = The Supermatter has Reached Critical Integrity Falure. Emergency Causality Destabilization Field has been Activated.
```

Comment:
instead of `The Supermatter` just pull the entity name as an argument and do `{CAPITALIZE(THE($entity))}`